.eslintrc.js

```diff
@@ -34,10 +34,10 @@ module.exports = {
   },
 
   extends: [
-    'prettier',
     'plugin:import/typescript',
     'plugin:react/recommended',
     'plugin:react-hooks/recommended',
+    'prettier',
```

Comment:
ESLint applies configs cascading, and `eslint-config-prettier` **always** must be last!